src/realm/node_header.hpp

```diff
@@ -405,6 +405,11 @@ class NodeHeader {
     // Setting the number of elements in the array(s). All encodings except Flex have one number of elements.
     static inline void set_num_elements(char* header, size_t num_elements, Encoding);
 
+    static inline size_t calc_size(size_t num_elements);
```

Comment:
I deleted these by mistake. There are needed and will be used by the integer compressor (next PRs).